docs/content/patterns/alz/Known-Issues.md

```diff
@@ -83,7 +87,7 @@ If you deployed AMBA just one time, you have 14 deployment instances
 The limit of 800 deployment for the given management group scope has been reached. More information can be found at [Management group limits](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#management-group-limits)
 
 ### Resolution
-To resolve this issue, follow the steps below:
+To resolve this issue, follow the following steps:
```

Comment:
follow the following. Maybe this works better?

```suggestion
To resolve this issue, follow these steps:
```

Reply:
This one is coming from Acrolinx spell and grammar check. We should not change it